rust/index/bindings.cpp

```diff
@@ -83,7 +91,7 @@ class Index
     {
         if (index_inited)
         {
-            std::runtime_error("Index already inited");
+            throw std::runtime_error("Index already inited");
```

Comment:
:'(

Reply:
These errors are all useless anyways, the calling code never lets this happen by construction.

Reply:
i'm still allowed to be sad